### ISSUE TITLE
Didn't see this in Package Control

### ISSUE THREAD

**Issue by excetara2** [sflip/SublimeMoveMore#1]
Did you ever add this to package control? I didn't see it is a package. Anyways it was exactly what I wanted and I slightly modified your code to improve it.

Cheers

**Comment by flipsi** [comment]
Thank you! 
I wanted to add it to Package Control, but my pull request was rejected (https://github.com/wbond/package_control_channel/pull/5064), because there is a similar plugin called LineJumper. 
However, I would be happy to get a pull request with your improvements!

**Comment by excetara2** [comment]
Yes I guess it make sense to use that is it has all your features. If you
add my lines of code I think they are essentially the same but uses more
the sublime text syntax.

Cheers
